src/styleProps/textProps.ts

```diff
@@ -7,37 +7,39 @@ const textTransformMap = {
 	none: 'none',
 };
 
+type TextTransformKey = keyof typeof textTransformMap;
+
 export const textPropsDef = [
 	{
 		styleProp: 'bold',
 		properties: ['fontWeight'],
-		getValue: bold => (bold ? 'bold' : 'normal'),
+		getValue: (bold: unknown) => (bold ? 'bold' : 'normal'),
 	},
 	{
 		styleProp: 'fontSize',
 		properties: ['fontSize'],
-		getValue: (size, theme) =>
+		getValue: (size: string, theme: any) =>
 			size ? theme.globals.typeScale[size] : undefined,
```

Comment:
I wonder if there's anything we can do to dynamically type-check these. Basically, `size` will be a string, but _what_ string depends on the keys available in `theme.globals.typeScale` (e.g. `"xs", "s", "m", "l"`), which can be different in any app, and is only determined at runtime. Actually, I suppose that answers my question, and we probably can't do this. 😅